tools/azsdk-cli/docs/specs/0-scenario-1.spec.md

```diff
@@ -379,41 +378,44 @@ Updating package metadata for Health Deidentification SDK...
 Package metadata updated for 5/5 languages
 ```
 
-### 4. Run PR Checks
+### 4. Run Validation Checks
 
-> Sample: will be changed when the spec for this tool is added
 
 **Command:**
 ```bash
-azsdk run-pr-checks --service healthdataaiservices --languages .NET,Java,JavaScript,Python,Go
+# Run all checks (default)
+azsdk package validate --package-path <path_to_sdk_package>/
+
+# Run specific check type
+azsdk package validate <check-type> --package-path <path_to_sdk_package>/
 ```
 
 **Options:**
-- `--service <name>`: Service name to check (required)
-- `--languages <list>`: Languages to check (default: all)
-- `--checks <list>`: Specific checks to run (build,test,lint,breaking-changes)
-- `--parallel`: Run checks in parallel
-- `--fail-fast`: Stop on first failure
+- `--package-path <path>`: Path to the specific SDK package directory (required)
+- `--fix`: Option to fix the errors reported (Milestone 2)
```

Comment:
Do we need a languages option if the user is only working on a single language or subset of languages? Or is the tool doing something smart to figure this out without the need of an extra option?

Reply:
it takes in a single path so it works on A language at a time, in the future we can expand this to take in a list